File: Modules/HumanResource/EmployeeProfile/EmployeeProfile/EmployeeType.cs
using System.ComponentModel;

namespace HRMSoftware.EmployeeProfile;
[Serenity.ComponentModel.EnumKey("HumanResource.EmployeeProfile.EmployeeType")]
public enum EmployeeType
{
    [Description("Local")]
    Local = 1,
    [Description("Foreigner")]
    Foreigner = 2
}

File: Modules/ServerTypes/EmployeeProfile/EmployeeType.ts
import { Decorators } from "@serenity-is/corelib";

export enum EmployeeType {
    Local = 1,
    Foreigner = 2
}
Decorators.registerEnumType(EmployeeType, 'HRMSoftware.EmployeeProfile.EmployeeType', 'HumanResource.EmployeeProfile.EmployeeType');